src/utilities/timeline.test.ts

```diff
@@ -1185,3 +1186,55 @@ test('getUniqueColorSchemeForXRangeLayer', () => {
   const existingScheme = (row2.layers[0] as XRangeLayer).colorScheme;
   expect(getUniqueColorSchemeForXRangeLayer(row2)).not.toBe(existingScheme);
 });
+
+describe('getTimeRangeAroundTime', () => {
+  //utility to convert ISO date to ms
+  function getTime(d: string): number {
```

Comment:
We actually already have a utility function for this in the `utilities/time.ts` called `convertUTCToMs`

Reply:
@duranb ah great! I can use that one.